code/controllers/UserController.php

```diff
@@ -17,8 +17,9 @@ public function index()
     {
         if (!Member::currentUserID()) {
             return $this->redirect('/Security/login');
-        }
-        return $this->renderWith(array('UserController', 'Page'));
+        } else if (UserInvitation::create()->canCreate(Member::CurrentUser()) == false){
```

Comment:
Just tested, It does work but has the implication that the below response is returned instead of the nicer "You don't have permission to access this. Login with another account if you have one" which `Security::permissionFailure();` returns
```
Action 'index' isn't allowed on class UserController.
```